doc/DeveloperGuide.md

```diff
@@ -88,6 +88,27 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+
+### Use case: Rename a tag
+
+**MSS**
+
+1. User requests to rename a tag.
+2. AddressBook requests for the new tag name.
+3. User enters a new tag name.
+4. AddressBook renames the tag. 
+5. AddressBook informs the User of the renaming.<br>
```

Comment:
Use cases should depict the back and forth interaction between the user and the system. For every step the user does, the system should do something in return (and vice-versa).